src/steth/scDai-adapters/SparkScDaiAdapter.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {WETH} from "solmate/tokens/WETH.sol";
+import {IPool} from "aave-v3/interfaces/IPool.sol";
+import {IPoolDataProvider} from "aave-v3/interfaces/IPoolDataProvider.sol";
+
+import {Constants as C} from "../../lib/Constants.sol";
+import {IAdapter} from "../IAdapter.sol";
+
+/**
+ * @title Aave v3 Lending Protocol Adapter
+ * @notice Facilitates lending and borrowing for the Aave v3 lending protocol
+ */
+contract SparkScDaiAdapter is IAdapter {
+    ERC20 constant dai = ERC20(C.SDAI);
+    WETH constant weth = WETH(payable(C.WETH));
```

Comment:
you can also make these public

Reply:
[done](https://github.com/lindy-labs/sandclock-contracts/pull/155/commits/84a09c8999e70e3d40d6dee80b4389d0bdbcedbc)